src/pages/LandingPage.js

```diff
@@ -67,7 +71,40 @@ const makeDocLinks = _.map(
 )
 
 const LandingPage = () => {
+  const { isSignedIn } = useStore(authStore)
+  const [billingProjects, setBillingProjects] = useState()
+  const signal = useCancellation()
+
+  const loadProjects = withErrorHandling(async error => {
+    const errorJSON = await error.json()
+    console.log(`Unable to load billing projects due to: ${errorJSON?.message}`)
```

Comment:
You can add this at the end of the line to suppress the lint warning.

`// eslint-disable-line no-console`